### PR TITLE
[Fix] NetworkManager binding to [::]

### PR DESCRIPTION
**Pull request by BaronGreenback**
Reported by @joshuaboniface.

[::] is now interpreted as being all ipv6 interfaces.
Updated testing packages to latest nuget version as not all tests were being detected.
Corrected a couple of spelling mistakes.

### CODE REVIEW

File: tests/Jellyfin.Networking.Tests/NetworkTesting/Jellyfin.Networking.Tests.csproj [hunk 1]
```diff
@@ -13,16 +13,16 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.8.0" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.9.0-preview-20201020-06" />
```
Comment (crobibero) [inline tests/Jellyfin.Networking.Tests/NetworkTesting/Jellyfin.Networking.Tests.csproj:16]: Unrelated changes?

Comment (BaronGreenback) [reply tests/Jellyfin.Networking.Tests/NetworkTesting/Jellyfin.Networking.Tests.csproj:16]: Did it on purpose - VS wasn't detecting all the tests. Found a resolution online to say upgrade testing packages. Did and it worked - so left it.

Can try to roll back?

Comment (crobibero) [reply tests/Jellyfin.Networking.Tests/NetworkTesting/Jellyfin.Networking.Tests.csproj:16]: Try just running `dotnet test` if VS doesn't detect all of the tests

**Review by crobibero — APPROVED** [review]

File: tests/Jellyfin.Networking.Tests/NetworkTesting/NetworkParseTests.cs [hunk 2]
```diff
@@ -169,18 +170,19 @@ public void InvalidAddressString(string address)
             "[]",
             "[]",
             "[]")]
+        [InlineData(
+            "192.158.1.2/16, localhost, fd23:184f:2029:0:3139:7386:67d7:d517,    !10.10.10.10",
```
Comment (Artiume) [inline tests/Jellyfin.Networking.Tests/NetworkTesting/NetworkParseTests.cs:174]: what's with 192.158? wouldnt we want 192.168?

Comment (BaronGreenback) [reply tests/Jellyfin.Networking.Tests/NetworkTesting/NetworkParseTests.cs:174]: These are just random ip addresses to check that the methods are working correctly, and that they can parse any ip address.

File: MediaBrowser.Common/Net/INetworkManager.cs [hunk 3]
```diff
@@ -192,9 +192,9 @@ public interface INetworkManager
         /// Parses an array of strings into a Collection{IPObject}.
         /// </summary>
         /// <param name="values">Values to parse.</param>
-        /// <param name="bracketed">When true, only include values in []. When false, ignore bracketed values.</param>
+        /// <param name="negated">When true, only include values in []. When false, ignore bracketed values.</param>
```
Comment (oddstr13) [inline MediaBrowser.Common/Net/INetworkManager.cs:195]: Documentation needs update

Comment (BaronGreenback) [reply MediaBrowser.Common/Net/INetworkManager.cs:195]: good catch

**Review by oddstr13 — APPROVED** [review]
